GVFS/GVFS/CommandLine/DiagnoseVerb.cs

```diff
@@ -132,12 +133,20 @@ protected override void Execute(GVFSEnlistment enlistment)
                         if (GVFSPlatform.Instance.UnderConstruction.SupportsGVFSUpgrade)
                         {
                             // upgrader
+                            this.CopyAllFiles(

```

Comment:
So now we'll have:
UpgradeDirectoryName/Logs
UpgradeDirectoryName/UpgradeLogs

We should only look at UpgradeLogs

Reply:
In the future we'll probably only look at `UpgradeDirectoryName/UpgraderLogs`.  

If an upgrade from the last GA release hit an issue, that would be in `UpgradeDirectoryName/Logs`.